compiler/src/dotty/tools/dotc/typer/UnusedUtil.scala

```diff
@@ -0,0 +1,32 @@
+package dotty.tools.dotc.typer
+
+import dotty.tools.dotc.ast.tpd._
+import dotty.tools.dotc.core.Contexts._
+import dotty.tools.dotc.core.Decorators._
+import dotty.tools.dotc.core.Symbols._
+import dotty.tools.dotc.core.Flags._
+
+/** Util methods for transformation of `unused` expressions
+ *
```

Comment:
Looks a bit small for a separate object. Can we find other methods that naturally go here? If not, maybe dissolve into another module?

Reply:
These two methods can be moved to `Applications`